File: 백준/Bronze/5596. 시험 점수/시험 점수.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String[] min = sc.nextLine().split(" ");
        int minResult = Integer.parseInt(min[0]) + Integer.parseInt(min[1]) + Integer.parseInt(min[2]) + Integer.parseInt(min[3]);

        String[] man = sc.nextLine().split(" ");
        int manResult = Integer.parseInt(man[0]) + Integer.parseInt(man[1]) + Integer.parseInt(man[2]) + Integer.parseInt(man[3]);

        if (minResult >= manResult) {
            System.out.println(minResult);
        } else {
            System.out.println(manResult);
        }

        sc.close();
    }
}

File: 백준/Bronze/5596. 시험 점수/시험 점수.cs
string[] min = Console.ReadLine().Split();

int minResult = int.Parse(min[0]) + int.Parse(min[1]) + int.Parse(min[2]) + int.Parse(min[3]);

string[] man = Console.ReadLine().Split();

int manResult = int.Parse(man[0]) + int.Parse(man[1]) + int.Parse(man[2]) + int.Parse(man[3]);

if (minResult >= manResult)
{
    Console.WriteLine(minResult);
}
    
else
    Console.WriteLine(manResult);